hardware/DomoticzHardware.h

```diff
@@ -81,7 +81,7 @@ class CDomoticzHardwareBase
 	void SendUVSensor(const int NodeID, const int ChildID, const int BatteryLevel, const float UVI, const std::string &defaultname);
 	void SendBlindSensor(const int NodeID, const int ChildID, const int BatteryLevel, const int Command, const std::string &defaultname);
 	void SendSoundSensor(const int NodeID, const int BatteryLevel, const int sLevel, const std::string &defaultname);
-	void SendAlertSensor(const int NodeID, const int BatteryLevel, const int alertLevel, const std::string &message, const std::string &defaultname);
+	void SendAlertSensor(const int NodeID, const int BatteryLevel, const int alertLevel, const std::string &message, const char * defaultname);
```

Comment:
I couldn't agree more, but the developper of SatelIntegra insisted in pull request 1356 to keep passing a NULL pointer:

szczukot 5 days ago Contributor
I don't want set name for this sensor.
Must be NULL instead "Alert"

But now that it is clear this is the root cause, I'd rather revert to std:string and replace the null pointer with an empty string.